SwiftyBluetooth/Source/Peripheral.swift

```diff
@@ -103,8 +103,8 @@ extension Peripheral {
     }
     
     /// Connect to the peripheral through Ble to our Central sharedInstance
-    public func connect(completion: @escaping ConnectPeripheralCallback) {
-        self.peripheralProxy.connect(completion)
+    public func connect(timeout: TimeInterval = 10, completion: @escaping ConnectPeripheralCallback) {
```

Comment:
Very good, not all swift devs use default value for parameters like this and many of them still write duplicate functions instead.

I would change it slightly to: "connect(withTimeout timeout: TimeInterval = 10"